src/category_theory/limits/limits.lean

```diff
@@ -302,6 +456,10 @@ def limit.lift (F : J ⥤ C) [has_limit F] (c : cone F) : c.X ⟶ limit F :=
   limit.lift F c ≫ limit.π F j = c.π.app j :=
 is_limit.fac _ c j
 
+@[simp] lemma limit.lift_π_assoc {F : J ⥤ C} [has_limit F] (c : cone F) (j : J) (W) (f : _ ⟶ W):
```

Comment:
This should use `reassoc`, I think.

Reply:
Fixed! (I think the PRs were running in parallel at first, and I forgot to do this once `@[reassoc]` was merged.)